domain/src/main/kotlin/com/krzykrucz/elesson/currentlesson/domain/startlesson/Types.kt

```diff
@@ -31,7 +32,17 @@ data class ScheduledLesson(val scheduledTime: ScheduledLessonStartTime,
                            val teacher: Teacher,
                            val className: ClassName)
 
-data class LessonHourNumber(val number: NaturalNumber)
+data class LessonHourNumber(val number: NaturalNumber) {
+    companion object {
+        private val SCHEDULE: Map<NaturalNumber, LocalTime> = mapOf(
+                NaturalNumber.ONE to LocalTime.parse("10:00:00"),
+                NaturalNumber.TWO to LocalTime.parse("10:55:00"),
+                NaturalNumber.THREE to LocalTime.parse("11:45:00")
+        )
+    }
+
+    fun getLessonScheduledStartTime(): LocalTime = SCHEDULE[this.number]!!
```

Comment:
It would be better to add a creator function in the companion object, validate the number inside this function and return Option<LessonHourNumber> from it

then when the getter is called we can be sure that the exception will not be thrown

constructor can also be made private in such case

Reply:
Done